packages/thumbprint-scss/mixins/type.scss

```diff
@@ -162,29 +162,29 @@
 // Text sizes
 
 @mixin tp-text-1--static($important: null) {
-    font-size: $tp-font__text__1__size $important;
-    line-height: $tp-font__text__1__line-height $important;
+    font-size: $tp-font__title__4__size $important;
+    line-height: 1.6 $important;
```

Comment:
The hardcoded values allow you to remove the deprecated tokens without breaking this component. Correct?

Reply:
Correct! There aren't any good equivalent tokens.